ui/nuxeo-video/nuxeo-video-info.js

```diff
@@ -0,0 +1,93 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior';
+
+{
+  /**
+  `nuxeo-video-info`
+  @group Nuxeo UI
+  @element nuxeo-video-info
+  */
+
+  class VideoInfo extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style>
+          .properties {
+            -moz-column-width: 25em; /* Firefox */
+            -webkit-column-width: 25em; /* webkit, Safari, Chrome */
+            column-width: 25em;
+            column-count: 3;
+            -moz-column-count: 3; /* Firefox */
+            -webkit-column-count: 3; /* webkit, Safari, Chrome */
+          }
+
+          .properties label {
+            width: 10em;
+            overflow: hidden;
+            text-overflow: ellipsis;
+          }
+
+          .item {
+            @apply --layout-horizontal;
+            @apply --layout-flex;
+          }
+
+          .item label {
+            color: var(--secondary-text-color);
```

Comment:
Should we define a default (fallback) color?

Reply:
any examples?

Reply:
One of the comments above (https://github.com/nuxeo/nuxeo-elements/pull/216#discussion_r394274401) contains an example 🙂